requirements.txt

```diff
@@ -1 +0,0 @@
-.
```

Comment:
This one needs to stay, as it serves as the proxy to setup.cfg...

Reply:
My bad. Let's get this fixed.